internal/datatransfer/monitor.go

```diff
@@ -282,8 +282,8 @@ func (m *Monitor) syncPath(source, target string) error {
 	log.Debug(logMessage)
 	err := m.fsSync.SyncPath(source, target)
 	if err != nil {
-		log.Errorf("error while %s", logMessage)
-		return fmt.Errorf("error while %s", logMessage)
+		log.Errorf("error while %s, %s", logMessage, err)
+		return fmt.Errorf("error while %s, %s", logMessage, err)
```

Comment:
To be consistent with other log messages,  can you please reword the log message as `error while %s; err: %s`

(See for example this [log message](https://github.com/project-flotta/flotta-device-worker/blob/6fb49625319fd4742cbb84f029ebca4c840bb90a/internal/datatransfer/monitor.go#L160))

Reply:
done